packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -40,9 +40,13 @@ export interface Memory {
 export interface PerformanceWithMemory extends IsomorphicPerformance {
 	readonly memory: Memory;
 }
+
 /**
  * Broad classifications to be applied to individual properties as they're prepared to be logged to telemetry.
- * Please do not modify existing entries for backwards compatibility.
+ *
+ * @privateRemarks Please do not modify existing entries for backwards compatibility.
```

Comment:
Slightly ambiguous otherwise (I first read it as "do not make changes whose purpose is to maintain back compat" which threw me off).
```suggestion
 * @privateRemarks Please do not modify existing entries, to maintain backwards compatibility.
```